### ISSUE TITLE
External link with strikethrough text not clickable

### ISSUE THREAD

**Issue by degart** [BoostIO/Boostnote#3550]
# Current behavior
External link with strikethrough text not opening.

![bug1_](https://user-images.githubusercontent.com/1138641/79505114-95edbc80-803c-11ea-9d88-6401a9d9d04e.gif)

# Expected behavior
The link should work similar as a link with normal text.

# Steps to reproduce

```
[clicable link](http://google.com)
[~~not clicable link~~](http://google.com)
```

# Environment

- Boostnote version: 0.15.2 
- OS version and name: macOS Mojave 10.14.1

**Comment by Flexo013** [comment]
Confirmed for Windows 10 with 0.15.3.

**Comment by Flexo013** [comment]
This is solved as of Boostnote Legacy version 0.16.0.

Thanks for your contribution @daiyam ❤️
